src/network/networkd-route.c

```diff
@@ -1743,7 +1743,7 @@ int config_parse_gateway(
                         return 0;
                 }
 
-                if (streq(rvalue, "_dhcp6")) {
+                if (streq(rvalue, "_ra")) {
```

Comment:
hmm, this is compat breakage, no? Or did we never release with this change?

maybe call this `_ipv6ra`?

(dunno, just wondering)

Reply:
ah, i see now, it was added in d306d1d0ca2e6559a598a6eb459f65f9377f909e only, which we haven#t released so far.